packages/devtools_app/lib/src/shared/server/_analytics_api.dart

```diff
@@ -71,15 +71,10 @@ Future<String?> getConsentMessage() async {
 
 /// Confirm with package:unified_analytics that the consent message
 /// has been shown to the user.
-Future<bool> confirmConsentMessageShown() async {
+Future<void> confirmConsentMessageShown() async {
```

Comment:
same comments here about naming. Maybe `markConsentMessageAsShown` if that is what this method does.